faqs/why-is-eth-switching-to-pos.md

```diff
@@ -1,7 +1,15 @@
 ---
 title: Why is ETH switching to Proof of Stake (Pos)?
-weight: 2.0
+weight: 2.1
```

Comment:
You can leave the weight at 2.0 since we're not adding an additional question here.